stackslib/src/chainstate/nakamoto/mod.rs

```diff
@@ -4275,6 +4278,46 @@ impl NakamotoChainState {
             .as_ref()
             .map(|rewards| rewards.reward_info.clone());
 
+        if do_not_advance {
+            // get burn block stats, for the transaction receipt
+            let (parent_burn_block_hash, parent_burn_block_height, parent_burn_block_timestamp) =
+                if block.is_first_mined() {
+                    (BurnchainHeaderHash([0; 32]), 0, 0)
```

Comment:
Is this ever reachable?

Reply:
Nope -- removed in https://github.com/stacks-network/stacks-core/pull/5346/commits/84950edfa160e6662b3cbd41a14522fcf2fb9ba7